app/script/calling/CallingRepository.js

```diff
@@ -1253,7 +1255,9 @@ z.calling.CallingRepository = class CallingRepository {
 
     return this.userRepository
       .get_user_by_id(userId)
-      .then(remoteUserEntity => this._createCall(callMessageEntity, remoteUserEntity))
+      .then(remoteUserEntity =>
+        this._createCall(callMessageEntity, remoteUserEntity, z.calling.enum.CALL_STATE.INCOMING)
+      )
```

Comment:
Let's stick to using explicit returns if it doesn't fit into a single line:

````
.then(remoteUserEntity => {
  return this._createCall(callMessageEntity, remoteUserEntity, z.calling.enum.CALL_STATE.INCOMING);
})
````

Reply:
:+1: